azooKeyMac/InputController/UserAction+getUserAction.swift

```diff
@@ -106,7 +106,13 @@ extension UserAction {
         case 48: // Tab
             return .tab
         case 49: // Space
-            return .space
+            switch (Config.TypeHalfSpace().value, event.modifierFlags.contains(.shift)) {
+            case (true, true), (false, false):
+                // 全角スペース
+                return .space(true)
+            default:
```

Comment:
(Bool, Bool)ならパターン数は多くないので、defaultではなく(true, false)と(false, true)を明示的に書いてしまいたいです。
ちなみに、「常に半角スペース」がONの場合もシフトを押しながらの半角スペース入力は全角になりますが、これは意図通りの設計ですか？
```suggestion
            case (true, false), (false, true):
```

Reply:
> ちなみに、「常に半角スペース」がONの場合もシフトを押しながらの半角スペース入力は全角になりますが、これは意図通りの設計ですか？

はい、ここは意図した通りです。
Google日本語入力（他のIMEがどうなのかわからないのですが）と動作感を合わせています。
設定の内容と違うよなぁというのはあるんですが、設定項目のわかりやすさを考えるとこうなるのかなと。悩ましいポイントです。

Reply:
理解です！